Parsers/ASimRegistryEvent/Parsers/vimRegistryEventMicrosoftWindowsEvent.yaml

```diff
@@ -12,7 +12,8 @@ References:
   Link: https://aka.ms/ASimRegistryEventDoc
 - Title: ASIM
   Link: https:/aka.ms/AboutASIM
-Description: ASIM Windows Event Registry Creation Event Parser (event number 4657)
+Description: |
+        This ASIM parser supports normalizing Microsoft Windows Event logs, produced by the Windows Forwarded Events connector, to the ASIM Registry Event normalized schema (event number 4657).
```

Comment:
-> "This ASIM parser supports normalizing Microsoft Windows events (event number 4657), produced by the Windows Forwarded Events connector, to the ASIM Registry Event normalized schema."


"